src/metabase/models/user_key_value.clj

```diff
@@ -0,0 +1,29 @@
+(ns metabase.models.user-key-value
+  (:require
+   [metabase.util.malli :as mu]
+   [methodical.core :as methodical]
+   [toucan2.core :as t2]))
+
+(methodical/defmethod t2/table-name :model/UserKeyValue [_model] :user_key_value)
+
+(doto :model/UserKeyValue
+  (derive :metabase/model)
+  (derive :hook/timestamped?))
+
+(mu/defn insert!
```

Comment:
What do you think about calling this `put!` instead, to avoid punning with `t2/insert!`?